src/types/meets.ts

```diff
@@ -14,6 +14,7 @@ export type Runner = {
   proposition_id_hash: string;
   barrier: number;
   signature: EcSignature;
+  bettingStatus: "Open" | "Scratched" | "LateScratched";
```

Comment:
inline with comment on BE pr -- could just be `status` rather than `bettingStatus` but no matter